kube/src/runtime/reflector.rs

```diff
@@ -59,10 +62,18 @@ where
             // local development needs listening for ctrl_c
             let ctrlc_fut = ctrl_c().fuse();
             // kubernetes apps need to listen for SIGTERM (30s warning)
-            use signal::unix::{signal, SignalKind}; // TODO: conditional compile
+            #[cfg(not(target_family = "windows"))]
```

Comment:
You should be able to create a new scope and conditionally compile that scope to make this easier to read:
```rust
 #[cfg(not(target_family = "windows"))]
 let sigterm_fut = {
  use signal::unix::{signal, SignalKind};
  let mut sigterm = signal(SignalKind::terminate()).unwrap();
  sigterm.recv().fuse()
};
```

Reply:
Sweet, I was wondering how I could do this

Reply:
So it looks like this doesn't work because `recv` borrows the receiver and then that borrow doesn't last outside the scope